app/helpers/document_helper.rb

```diff
@@ -217,8 +217,11 @@ def part_of_metadata(document, policies = [], sector_tag_finder = nil)
       part_of += array_of_links_to_topical_events(document.topical_events)
     end
 
-    if sector_tag_finder && (all_sectors = sector_tag_finder.sectors_and_subsectors).any?
-      part_of += array_of_links_to_sectors(all_sectors)
+    if sector_tag_finder && (tagged_topics = sector_tag_finder.topics).any?
+      links_to_topics = tagged_topics.map do |topic|
+        link_to topic.title, topic.web_url, class: 'sector-link'
+      end
+      part_of += links_to_topics
```

Comment:
Wouldn't it be better to use `<<` instead of `+=` ?

http://ruby-doc.org/core-1.9.3/Array.html#method-i-2B

> Concatenation—Returns a new array built by concatenating the two arrays together to produce a third array.

http://ruby-doc.org/core-1.9.3/Array.html#method-i-3C-3C

> Append—Pushes the given object on to the end of this array. This expression returns the array itself, so several appends may be chained together.